staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_type.go

```diff
@@ -771,7 +771,7 @@ func (c *$.type|privatePlural$) Watch(opts $.ListOptions|raw$) $.aggregatedWatch
 			VersionedParams(&opts, $.schemeParameterCodec|raw$).
 			Timeout(timeout).
 			Watch()
-		if err != nil && opts.AllowPartialWatch {
+		if err != nil && opts.AllowPartialWatch && $.errorForbidden|raw$(err) {
 			// watch error was not returned properly in error message. Skip when partial watch is allowed
 			$.klogInfof|raw$(6).Infof("Watch error for partial watch %v. options [%+v]", err, opts)
```

Comment:
Is it the expected behavior? An error is logged as Info with a very low level of 6.

Besides, the log info is not clear. Add more info about the resources,  tenant/space/type,  in the log message.

Reply:
This log is intentionally used in level 6, which cannot be used in level 6. This is a valid case and thus not an error. Also, the code is generated by code-gen, it is used in cases that might not have tenant and namespace, thus it is complicated to add log for tenant/namespace. I don't think it is worth to add due to code complexity and usage.